addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  _updateModel(model, infinityModel) {
```

Comment:
Will un-underscore the method name. I'm not sure about the compatibility of `pushObjects` with the latest Ember data, but that doesn't seem relevant to this change (since I'm just moving code around).